align/constGap_lowMem3.go

```diff
@@ -0,0 +1,225 @@
+package align
+
+import (
+	"github.com/vertgenlab/gonomics/dna"
+	"log"
+	"math"
+)
+
+func ConstGap(alpha []dna.Base, beta []dna.Base, scores [][]int64, gapPen int64) (int64, []Cigar) {
+
+	var checkersize int
+	checkersize = 10000 //TODO: make checkersize an cmd option; checkersize_i and checkersize_j
```

Comment:
Did you initialize at 1000 for a reason or is it just an arbitrary large number?